src/lib/y2partitioner/actions/create_partition_table.rb

```diff
@@ -0,0 +1,98 @@
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2partitioner/actions/transaction_wizard"
+require "y2partitioner/actions/controllers"
+require "y2partitioner/dialogs"
+
+module Y2Partitioner
+  module Actions
+    # Action for creating a new partition table
+    class CreatePartitionTable < TransactionWizard
```

Comment:
Right now this might be the case, but if we need to add one more step in the workflow because we need to offer more choices to the user that might no longer be true.